### PR TITLE
T1621

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/f83c0c06da8d0d2/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/f83c0c06da8d0d2/output.txt

Total script time: 0.59 mins

Server is running on http://pr562.butyr.org/templates/test.html

File: templates/supported/shared/plugins/speak/popcorn.speak.js [hunk 1]
```diff
@@ -189,6 +189,12 @@
               var p = popcornInstances[ event.data.popcornID ],
                   options = p.getTrackEvent( event.data.optionsID );
 
+              // Bail if options object has vanished since this
+              // request was originally made to build the speech.
+              if( !options ){
```
Comment (mjschranz) [inline templates/supported/shared/plugins/speak/popcorn.speak.js:194]: Needs space padding before and after the round brackets

**Pull request merged**